packages/flutter_tools/test/general.shard/base/logger_test.dart

```diff
@@ -669,50 +736,73 @@ void main() {
       // Verify that stopping or canceling multiple times throws.
       expect(() { summaryStatus.stop(); }, throwsA(isInstanceOf<AssertionError>()));
       expect(() { summaryStatus.cancel(); }, throwsA(isInstanceOf<AssertionError>()));
-    }, overrides: <Type, Generator>{Stdio: () => mockStdio, Platform: _kNoAnsiPlatform});
+    }, overrides: <Type, Generator>{
+      Platform: () => _kNoAnsiPlatform,
+    });
 
-    testUsingContext('sequential startProgress calls with StdoutLogger', () async {
-      final Logger logger = context.get<Logger>();
-      logger.startProgress('AAA', timeout: timeoutConfiguration.fastOperation)..stop();
-      logger.startProgress('BBB', timeout: timeoutConfiguration.fastOperation)..stop();
+    // Status still uses Stopwatch from context.
```

Comment:
TODO(jonahwilliams): Change to testWithoutContext when Status no longer uses the Stopwatch from the context.